README.md

```diff
@@ -119,6 +122,34 @@ associated to `linux` and `apache-servers` subscriptions.
   }
 ```
 
+### Manage Windows Agent
+
```

Comment:
Should document which versions are supported

Reply:
Added that support starts with 5.7.0